go.mod

```diff
@@ -303,6 +303,7 @@ replace (
 	// fix potential security issue(CVE-2020-26160) introduced by indirect dependency.
 	github.com/dgrijalva/jwt-go => github.com/form3tech-oss/jwt-go v3.2.6-0.20210809144907-32ab6a8243d7+incompatible
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
+	github.com/pingcap/kvproto => github.com/nolouch/kvproto v0.0.0-20230630091705-834c1b8230e1
```

Comment:
what's the PR in kvproto that will remove this replace? I want to subscribe on it

Reply:
https://github.com/pingcap/kvproto/pull/1142 @lance6716

Reply:
after discussing, this kvproto's modification was abandoned. For now, the relevant modifications are
https://github.com/tikv/client-go/pull/868 :)